src/errors.ts

```diff
@@ -38,15 +49,10 @@ class ConnectionError extends WidgetError {
   }
 }
 
-export class SwapError extends WidgetError {
+export class SwapError extends DismissableWidgetError {
   constructor(config: WidgetErrorConfig) {
-    super({
-      ...config,
-      header: config.header ?? DEFAULT_DISMISSABLE_ERROR_HEADER,
-      action: config.action ?? DEFAULT_DISMISSABLE_ERROR_ACTION,
-    })
+    super(config)
```

Comment:
If you use

    super({ header: t`Swap Error`, ...config })

you can omit the header from all other callsites.